src/main/java/uk/gov/ons/registers/cdap/service/TableColumns/CompanyHouseTable.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.ons.registers.cdap.service.TableColumns;
```

Comment:
By [convention](https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html), package name should be all lower case:

> Package names are written in all lower case to avoid conflict with the names of classes or interfaces.